### ISSUE TITLE
Bug: Misleading 'Job Sessions after filter: 0' debug log in concurrency calculation

### ISSUE THREAD

**Issue by comnam90** [VeeamHub/veeam-healthcheck#113]
## Summary

When running with `/debug`, the log emits a spurious and alarming line:

```
DEBUG   Job Sessions TOTAL = 62
DEBUG   Report interval: 7
DEBUG   Job Sessions after filter: 0
```

This appears multiple times (once per concurrency call) and implies all sessions are being discarded before concurrency analysis. They are not — the concurrency output is correct.

## Root Cause

In `CDataFormer.JobConcurrency()` (around line 923), a debug-only intermediate variable `v` is computed with the **wrong sign** on the date offset:

```csharp
// Line 923 — debug variable only, never used for computation
var v = CGlobals.DtParser.JobSessions.Where(c => c.CreationTime >= CGlobals.GetToolStart.AddDays(CGlobals.ReportDays));
// ^^^ +ReportDays pushes the cutoff INTO THE FUTURE — always returns 0

// Line 929 — actual filter used for concurrency, correct
trimmedSessionInfo = CGlobals.DtParser.JobSessions.Where(c => c.CreationTime >= CGlobals.GetToolStart.AddDays(-CGlobals.ReportDays)).ToList();
```

Because `+ReportDays` (e.g. `+7`) projects the cutoff date into the future, every session in the CSV pre-dates it, so `v.Count()` is always 0. The variable `v` is then printed in the debug log and discarded — it plays no part in the concurrency calculation, which uses the correctly-signed `trimmedSessionInfo`.

## Impact

**No impact on report output.** The concurrency data in the HTML report is correct. The only effect is a confusing debug log that suggests a data loss that did not occur.

## Repro

Run with `/debug` against any VBR server. The line `Job Sessions after filter: 0` will appear regardless of how many sessions exist.

## Suggested Fix

Either:

1. Fix the sign in the debug expression to match the real filter:
   ```csharp
   var v = CGlobals.DtParser.JobSessions.Where(c => c.CreationTime >= CGlobals.GetToolStart.AddDays(-CGlobals.ReportDays));
   ```

2. Or remove the dead variable `v` and its debug block entirely — `trimmedSessionInfo.Count` is already logged on the next line via the `"Loaded X Job Sessions for Concurrency..."` INFO message, which correctly reflects the actual filter result.

## Environment

- vHC Version: 3.0.1.101
- VBR Version: 13.0.1.1071
- Run mode: CLI (`/run /debug`)

**Comment by adamcongdon** [comment]
Fixed in v3.0.1.118 — removed the dead variable with the sign error (`AddDays(+ReportDays)`) that always produced a future cutoff date, logging 0 sessions. The correct data query already used `-ReportDays`.
